src/shared/utils/getDotnetInfo.ts

```diff
@@ -25,6 +25,7 @@ export async function getDotnetInfo(dotNetCliPaths: string[]): Promise<DotnetInf
 
```

Comment:
This file uses --info. --info is not meant for automation and does not have a guaranteed format. The recommendation is to use --list-runtimes and --list-sdks. If there is data you need that is not avaialble in those existing commands, please let us know and we can look into additional stable, machine readable options that are not --info.

CC @elinor-fung @vitek-karas since the work would come out of the host to add new information/commands potentially.

Reply:
I can probably replace the version lookup using the --list-sdks command with some work.  However we also read the runtimeid (for the debugger) and the architecture (for this PR) which don't appear to be in either of those commands.

Generally I would like to avoid needing to parse dotnet info here as well - and once the new version of the sdk acquisition extension is available I will likely be able to remove the part of the code I'm adding here.  I'm not so sure about the other use cases like the debugger (cc @gregg-miskelly).

Reply:
For the debugger, we are trying to check the version and architecture of the globally installed .NET, as we need the one the application is going to run on. If we want to switch the target app to run on the version of .NET that the acquisition extension downloads, we could probably get rid of the check. But `dotnet --list-runtimes` and `dotnet --list-sdks` don't have architecture information, so those commands aren't useful enough.

Reply:
Thinking about it more: as long as we provide a way to specify a .NET SDK in the extension, we probably are going to need a way to check the architecture.